### PR TITLE
Implement Ingress configuration on Helm Char

### LINKED ISSUE #27: Have an Ingress configuration on Helm Chart
Can be usefull to have a deploy option using Ingress instead of node port.
It's a bit more expensive on Cloud providers but is ~the right~ an elegant solution. 😸

### PR DESCRIPTION
**Pull request by eher-werkspot**
Fixes #27

### CODE REVIEW

**Review by malukenho — APPROVED** [review]

**Comment by malukenho** [comment]
@eher-werkspot You missed the signature for commits!

**Review by agustingomes — APPROVED** [review]

File: helm/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{{ if $.Values.ingress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Release.Name }}
+  {{- if $.Values.ingress.annotations }}
+  annotations: {{- $.Values.ingress.annotations | toYaml | nindent 4 }}
+  {{- end }}
+  labels:
+    turboEnigma: {{ .Release.Name }}
```
Comment (renan) [inline helm/templates/ingress.yaml:10]: Is this a good moment to change those labels to something more structured, like this https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ ?

So these labels could become `app.kubernetes.io/name: turbo-enigma` or event `app.kubernetes.io/name: {{ $.Chart.Name }}`

Comment (renan) [reply helm/templates/ingress.yaml:10]: PS: Changing the labels will force the recreation of the Deployment as some of these fields are immutable.

Comment (eher-werkspot) [reply helm/templates/ingress.yaml:10]: I think it can go in another merge request.

It's good to have an issue for it to keep track of the why.

**Review by EHER — APPROVED** [review]

**Review by EHER — COMMENTED** [review]
The commit must be signed.

File: helm/templates/ingress.yaml (hunk 1 above)
Comment (EHER) [reply helm/templates/ingress.yaml:10]: Issue created here https://github.com/codelicia/turbo-enigma/issues/32

**Review by EHER — APPROVED** [review]

**Pull request merged**